Exercises/4-count-types.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
-
-const countTypesInArray = null;
+const countTypesInArray = arr => {
+  const obj = {};
+  for (const a of arr) {
+    const type = typeof a;
+    obj[type] = obj[type] ? ++obj[type] : 1;
```

Comment:
Use additional identifiers (variables) and try to rewrite this with if-statement to improve readability.